Makefile

```diff
@@ -141,6 +127,12 @@ tailscale-test.apk: gradle-dependencies
 	(cd android && ./gradlew assembleApplicationTestAndroidTest)
 	install -C ./android/build/outputs/apk/androidTest/applicationTest/android-applicationTest-androidTest.apk $@
 
+tailscale.version: go.mod go.sum .git/HEAD
```

Comment:
does having a git file as a dep here cause problems for people who download a release tarball from github without the `.git` directory?

Reply:
I believe you can make this optional with `$(wildcard .git/HEAD)`, which will expand to the empty string if the file is not present. e.g.:
```
tailscale.version: go.mod go.sum $(wildcard .git/HEAD)
```